app/templates/components/explore/side-bar.hbs

```diff
@@ -71,13 +74,15 @@
             {{Category.name}}
           </a>
           {{#if (eq category Category.name)}}
-            {{#each Category.subTopics as |subCategory|}}
-              <a href="#"
-                class="link inner item {{if (eq sub_category subCategory.slug) 'active'}}"
-                {{action 'selectCategory' Category.name subCategory.slug}}>
-                {{subCategory.name}}
-              </a>
-            {{/each}}
+            <div class="ui form">
+              <div class="grouped fields">
+                {{#each Category.subTopics as |subCategory|}}
+                  <div class="field inner item">
+                    {{ui-checkbox label=subCategory.name class='toggle' checked=(if (eq sub_category subCategory.slug) 'active') onChange=(action 'selectCategory' Category.name subCategory.slug)}}
```

Comment:
Use checkbox rather than toggle 
<img width="225" alt="Screenshot 2019-07-15 at 4 13 54 AM" src="https://user-images.githubusercontent.com/44091822/61190250-2286db00-a6b7-11e9-81ab-c67edc702f17.png">